### PR TITLE
Correct spawn behavior of Lumber Jack - he does not spawn with enmity

### PR DESCRIPTION
**Pull request by TeoTwawki**
Move far enough away before the Weeping Willows dead body fades and Lumber Jack doesn't even notice you are there, will wander around till you engage, catch agro from walking to close in his line of sight, or his 10 min despawn timer is up. so no `updateEnmity()` needed here.

### CODE REVIEW

File: scripts/zones/Batallia_Downs/mobs/Weeping_Willow.lua [hunk 1]
```diff
@@ -5,17 +5,20 @@
...
 
 function onMobDeath(mob, player, isKiller)
-    local JACK = mob:getID()+6;
-    SpawnMob(JACK):updateEnmity(player);
+end
+
+function onMobDespawn(mob)
+    local JACK = mob:getID()+6
+    SpawnMob(JACK)
     GetMobByID(JACK):setPos( mob:getXPos(), mob:getYPos(), mob:getZPos(), 0);
```
Comment (zynjec) [inline scripts/zones/Batallia_Downs/mobs/Weeping_Willow.lua]: setSpawn, use it or lose it sister.

Comment (TeoTwawki) [reply scripts/zones/Batallia_Downs/mobs/Weeping_Willow.lua]: breaks behavior of him walking back up the cliff if pulled down so I wasn't sure I should make that change (I actually did then reverted it 2nd guessing myself). to get him 100% retail he should anchor to to where willow died *except* when willow dies down the cliff side at which point he goes back to the area where willow spawned at.

Still want me to switch it?

Comment (takhlaq) [reply scripts/zones/Batallia_Downs/mobs/Weeping_Willow.lua]: leave a comment in the script for now with retail behaviour vs current

Comment (TeoTwawki) [reply scripts/zones/Batallia_Downs/mobs/Weeping_Willow.lua]: done

**Pull request merged**